cluster-provision/k8s-fedora/1.17.0/provision.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
cluster-provision/k8s/1.17.0/fedora/...

Reply:
I structured the directories that way in order to keep that same structure as k8s directory,
then we could add more directories for other k8s versions.